hack/verify-staging-meta-files.sh

```diff
@@ -36,8 +34,8 @@ exceptions=(
 )
 
 RESULT=0
-for repo in ${staging_repos[@]}; do
-  for filename in ${expected_filenames[@]}; do
+for repo in ${KUBE_ROOT}/staging/src/k8s.io/*; do
```

Comment:
I think this is not quite equivalent.
```console
$ staging_repos=($(ls "staging/src/k8s.io/"))
$ echo ${staging_repos[@]}
api apiextensions-apiserver apimachinery apiserver client-go cli-runtime cloud-provider cluster-bootstrap code-generator component-base csi-api csi-translation-lib kube-aggregator kube-controller-manager kubelet kube-proxy kube-scheduler metrics node-api sample-apiserver sample-cli-plugin sample-controller
```
vs
```
$ for repo in staging/src/k8s.io/*; do echo $repo; done
staging/src/k8s.io/api
staging/src/k8s.io/apiextensions-apiserver
staging/src/k8s.io/apimachinery
staging/src/k8s.io/apiserver
staging/src/k8s.io/client-go
staging/src/k8s.io/cli-runtime
staging/src/k8s.io/cloud-provider
staging/src/k8s.io/cluster-bootstrap
staging/src/k8s.io/code-generator
staging/src/k8s.io/component-base
staging/src/k8s.io/csi-api
staging/src/k8s.io/csi-translation-lib
staging/src/k8s.io/kube-aggregator
staging/src/k8s.io/kube-controller-manager
staging/src/k8s.io/kubelet
staging/src/k8s.io/kube-proxy
staging/src/k8s.io/kube-scheduler
staging/src/k8s.io/metrics
staging/src/k8s.io/node-api
staging/src/k8s.io/sample-apiserver
staging/src/k8s.io/sample-cli-plugin
staging/src/k8s.io/sample-controller
```

(note the prefix added when using the glob)

Reply:
@ixdy You're right! It is not quite equivalent. I changed it back!